docs/topic-guides/effective-distributed-training.txt

```diff
@@ -22,14 +22,14 @@ How Distributed Training in Determined Works
 --------------------------------------------
 
 Distributed training in Determined utilizes data-parallelism. Data-parallelism for deep-learning
-consists of a set workers, where each worker is assigned to a unique compute accelerator such as a GPU or a TPU.
+consists of a set of workers, where each worker is assigned to a unique compute accelerator such as a GPU or a TPU.
 Each workers maintains a copy of the model parameters (weights that are being trained),
 which is synchronized across all the workers at the start of training.
 
 After initialization is completed, distributed training in Determined follows a loop where:
 
-#. Every worker performs a forward and backwards pass on a unique mini-batch of data.
-#. As the result of the backwards pass, every worker generates a unique set of updates to the solution state based on the data it processed.
+#. Every worker performs a forward and backward pass on a unique mini-batch of data.
+#. As the result of the backward pass, every worker generates a unique set of updates to the solution state based on the data it processed.
```

Comment:
non-blocking: could we change `solution state`-> `model parameters`, to make it more consistent with previous text.